pkg/cache/v3/snapshot_test.go

```diff
@@ -98,7 +98,7 @@ func TestScopedRouteListenerWithScopedRouteOnlyIsInconsistent(t *testing.T) {
 func TestScopedRouteListenerWithScopedRouteAndRouteIsConsistent(t *testing.T) {
 	snap, _ := cache.NewSnapshot(version, map[rsrc.Type][]types.Resource{
 		rsrc.ListenerType: {
-			resource.MakeScopedRouteHTTPListener(resource.Xds, "listener0", 80, "scopedRoute0"),
+			resource.MakeScopedRouteHTTPListener(resource.Xds, "listener0", 80),
```

Comment:
I guess now, is it possible to have Scoped Routes. But have no HTTP Listeners in the filter chain using SRDS?

Reply:
Yeh, the reverse is also possible. I thought about implementing that, but balked at having to iterate and unpack the filters a second time and make a special case check for scoped routes. WDYT @alecholmez, is it worth it?

Reply:
This is probably good for now, let's definitely note and potentially make an issue if that's something worth implementing down the road